api/v1beta2/ibmvpcmachine_types.go

```diff
@@ -36,10 +39,23 @@ type IBMVPCMachineSpec struct {
 	// Name of the instance.
 	Name string `json:"name,omitempty"`
 
+	// CatalogOffering is the Catalog Offering OS image which would be installed on the instance.
+	// An OfferingCRN or VersionCRN is required, the PlanCRN is optional.
+	// +optional
+	CatalogOffering *IBMCloudCatalogOffering `json:"catalogOffering,omitempty"`
+
+	// DedicatedHost is the name of the underlying provisioned host in the VPC on which the instance will be created.
+	// +optional
+	DedicatedHost *VPCResource `json:"dedicatedHost,omitempty"`
```

Comment:
For optional field for our better understanding lets mention what will happen when the field is omited.